src/SFA.DAS.AssessorService.Web/Views/Home/Index.cshtml

```diff
@@ -4,7 +4,7 @@
     ViewData["Title"] = "Apprenticeship assessment service";
     ViewBag.ZenDeskLabel = "ass-start-page";
     Layout = "_Layout";
-    var actionName = Model.UseGovSignIn ? "UpdateAnAccount" : "CreateAnAccount";
```

Comment:
This a link to CreateAnAccount being removed - could that be the only call to CreateAnAccount if so see the comment above about the Post